api/server/router/network/network_routes.go

```diff
@@ -85,6 +86,15 @@ func (n *networkRouter) getNetwork(ctx context.Context, w http.ResponseWriter, r
 	}
 
 	term := vars["id"]
+	var (
+		verbose bool
+		err     error
+	)
+	if v := r.URL.Query().Get("verbose"); v != "" {
+		if verbose, err = strconv.ParseBool(v); err != nil {
+			verbose = true
+		}
```

Comment:
Agreed. The current check is wrong. It should have been `err == nil` to set the verbose flag. Will fix it.